src/requests/adapters.py

```diff
@@ -154,6 +154,8 @@ def __init__(
 
         self.init_poolmanager(pool_connections, pool_maxsize, block=pool_block)
 
+        self.urllib3_response_options = {}
```

Comment:
If you want this to survive pickling you need to add it to `__attrs__` above, I suspect people will expect this to be picked appropriately